src/ObjectTypeComposer.ts

```diff
@@ -1884,9 +1884,8 @@ export class ObjectTypeComposer<TSource = any, TContext = any> {
       return catchErrors
```

Comment:
Just add `catchErrors: false` property and everything should work as you want  😉
```diff
UserTC.addRelation('taskss', {
    resolver: () => TaskQuery.taskMany,
    prepareArgs: {
        filter: (source) => ({
        user: source._id,
        })
      },
      projection: { user: 1},
+    catchErrors: false
  });
```